test.sh

```diff
@@ -3,7 +3,7 @@ set -e
 
 [[ "$ETH_RPC_URL" && "$(seth chain)" == "ethlive"  ]] || { echo "Please set a mainnet ETH_RPC_URL"; exit 1;  }
 
-SOLC_FLAGS="--optimize --optimize-runs 1" dapp --use solc:0.6.11 build
+SOLC_FLAGS="--optimize --optimize-runs 1" dapp --use solc:0.6.12 build
```

Comment:
Change this line to:
```
DAPP_BUILD_OPTIMIZE=1 DAPP_BUILD_OPTIMIZE_RUNS=1 dapp --use solc:0.6.12 build
```

For compatibility with newer dapp.

Reply:
Done!